Packs/MaintenanceUpdateNotifications/Playbooks/playbook-XSOAR_8_Maintenance_Update_Notification.yml

```diff
@@ -0,0 +1,440 @@
+                      iscontext: true
+                      value:
+                        simple: incident.xsoarcloudregion
+                - - ignorecase: true
+                    left:
+                      iscontext: true
+                      value:
+                        simple: RssFeed.title
+                    operator: containsGeneral
+                    right:
+                      value:
+                        simple: America Government
+                root: RssFeed
+          operator: isNotEmpty
+      label: America Government
+    - condition:
+      - - ignorecase: true
+          left:
+            iscontext: true
+            value:
+              complex:
+                filters:
+                - - ignorecase: true
+                    left:
+                      iscontext: true
+                      value:
+                        simple: RssFeed.title
+                    operator: containsGeneral
+                    right:
+                      value:
+                        simple: XSOAR
+                - - ignorecase: true
+                    left:
+                      iscontext: true
+                      value:
+                        simple: RssFeed.title
+                    operator: notContainsGeneral
+                    right:
+                      iscontext: true
+                      value:
+                        simple: incident.xsoarcloudregion
+                root: RssFeed
+          operator: isNotEmpty
+      label: Generic XSOAR Notifications
+    continueonerrortype: ''
+    id: '7'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - '4'
+      America Government:
+      - '8'
+      Generic XSOAR Notifications:
+      - '9'
+      Non America Government:
+      - '5'
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ''
+      id: f3028d3e-73a1-460b-8a54-41348e56a35c
+      iscommand: false
+      name: Filter by Region Group
+      type: condition
+      version: -1
+    taskid: f3028d3e-73a1-460b-8a54-41348e56a35c
+    timertriggers: []
+    type: condition
+    view: "{\n  \"position\": {\n    \"x\": 695,\n    \"y\": 370\n  }\n}"
+  '8':
+    continueonerrortype: ''
+    id: '8'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ''
+      forEach: true
+      iscommand: false
+      max: 100
+      wait: 1
+    nexttasks:
+      '#none#':
+      - '6'
+    note: false
+    quietmode: 0
+    scriptarguments:
+      MaintenanceDetails:
+        complex:
+          filters:
+          - - ignorecase: true
+              left:
+                iscontext: true
+                value:
+                  simple: RssFeed.title
+              operator: containsGeneral
+              right:
+                value:
+                  simple: XSOAR 8
```

Comment:
I saw that in the other conditions you've mentioned just `XSOAR` without the versioning. Seems like this should be also only XSOAR ( avoiding confusion in the future if we will have xsoar 9 :) ) 
WDYT?

Reply:
Fixed. Will be pushed to PR soon.